### PR TITLE
Newsletter Mail Feature with models

### PR DESCRIPTION
**Pull request by mayank2393**

### CODE REVIEW

File: server/src/controllers/newsletter.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export const mailer= async (req,res)=>{
+    const users=await User.find({newsletterIsSubscribed:true}); // this is an array of objects
+    const mails=users.map(user=>user.email); // this is an array of emails
+
+    const subject=req.body.subject;
+    const HTML=req.body.html_body;
+
+    mails.forEach(async(mail)=>{
+      await mailSender(mail,subject,HTML);
+    })
```
Comment (Himasnhu-AT) [inline server/src/controllers/newsletter.js]: Send it as BCC, don't send them in a loop

File: server/src/router/newsletter.js
Comment (Himasnhu-AT) [inline server/src/router/newsletter.js:1]: Cleanup file

Comment (Himasnhu-AT) [reply server/src/router/newsletter.js:1]: i mean remove unnecessary comments, lint it

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
Resolve following issues

**Comment by Himasnhu-AT** [comment]
@mayank2393 @ArnavSharma005, no need to close pr, just commit int branch and push, it'll automatically combine in this only

File: server/src/router/newsletter.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+//const controller=require('../controllers/news')
+
+// import controller from '../controllers/newsletter'
+import {mailer} from '../controllers/newsletter.js'
+
+// a get method to get all users with newsletterIsSubscribed as true
+router.get('/',async (req,res)=>{
+    
+    res.send( await User.find({newsletterIsSubscribed:true}));
+})
```
[the remark this answers is not in the document]
Comment (Himasnhu-AT) [reply server/src/router/newsletter.js:8]: Implemented on later stage

**Comment by Debatreya** [comment]
@Himasnhu-AT isse aj merge krde
Bakio o ko models chahiye honge

**Review by Himasnhu-AT — APPROVED** [review]

**Comment by Himasnhu-AT** [comment]
@Debatreya hogya

**Pull request merged**